components/cop_datatype/src/def/field_type.rs

```diff
@@ -308,3 +308,19 @@ impl FieldTypeAccessor for ColumnInfo {
         self as &mut dyn FieldTypeAccessor
     }
 }
+
+impl Into<FieldType> for FieldTypeTp {
```

Comment:
Could this be implemented as `impl From<FieldTypeTp> for FieldType`?

https://rust-lang-nursery.github.io/api-guidelines/interoperability.html#c-conv-traits

Reply:
Nop, because FieldType is a protobuf generated structure.